vue/components/ui/molecules/table/table.custom.stories.js

```diff
@@ -59,5 +61,32 @@ storiesOf("Molecules", module).add("Table Custom", () => ({
                 </tr>
             </template>
         </table-ripe>
+
+        <h2>Table 2 - Specific row override:</h2>
+        <table-ripe
+                class="table"
+                v-bind:columns="mockColumns"
+                v-bind:items="mockItems"
+            >
+            <template v-slot:item-2="{ item, index }">

```

Comment:
I admit I wasn't clear, but what I meant was being able to use `item-user` or `item-system` :), changing an entire column template instead of a single cell.